python/sauron-core/tests/test_classifier.py

```
@@ -0,0 +1,157 @@
+import pytest
+import numpy as np
+from core.analysis.classifier import AIImageClassifier
+
+def test_classifier_initialization():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_classifier_initialization`

Add return type annotation: `None`

(ANN201)